html/forms/html-form-structure/payment-form.html

```diff
@@ -81,9 +81,9 @@ <h2>Payment information</h2>
               <label for="date">
                 <span>Expiration date:</span>
                 <strong><abbr title="required">*</abbr></strong>
-                <em>formatted as mm/dd/yyyy</em>
+                <em>formatted as MM/YY</em>
               </label>
-              <input type="date" id="date" name="expiration">
+              <input type="text" id="date" name="expiration" placeholder="MM/YY">
```

Comment:
```suggestion
              <input type="text" id="date" name="expiration">
```

We don't need the placeholder here, as the format is stated in the label. Also it should reflect the example on the corresponding mdn page 👍

Reply:
Ok, but the format is wrong for card expiry date format, if we don't use the placeholder then how should i approach the problem, i can make the format mm/yy, using JavaScript